std/array.d

```diff
@@ -3927,3 +3927,214 @@ unittest
     assert(appS.data == "hellow");
     assert(appA.data == "hellow");
 }
+
+/++
+Constructs a static array from `a`.
+The type of elements can be specified implicitly (`auto a = [1,2].asStatic;` of type int[2])
+or explicitly (`auto a = [1,2].asStatic!float` of type float[2]). When `a` is a range,
```

Comment:
Inconsistency: you use once a semi-colon and in the next example you don't. I prefer it without the semi-colon, but I don't care as long as it's consistent.

Reply:
done